mcc/ttl/main.go

```diff
@@ -132,11 +133,13 @@ func UpsertResourceRecordSetTTL(
 		log.Panic(err.Error())
 	}
 	// Submit batch changes
-	changeResponse, err = svc.ChangeResourceRecordSets(changeRRSInput)
+	if !dryrun {
+		changeResponse, err = svc.ChangeResourceRecordSets(changeRRSInput)
```

Comment:
Right now, if it's a dryrun run `changeResponse` is nil. As the return value from the function isn't used afterwards before the end of the program (if dryrun there is no execution of `WaitForChangeToComplete`) I think it's fine for now. Nonetheless it provides dangerous side-effects if we try to use it in the future so we will need to fix/improve this behaviour before extending the tool further.